src/sections/Community/index.js

```diff
@@ -17,11 +17,11 @@ import { graphql, useStaticQuery } from "gatsby";
 import { getImage } from "gatsby-plugin-image";
 import { BgImage } from "gbimage-bridge";
 
-const CommunityMember = "./Community-pictures/Lee Calcote and Oliver Gould - CTO of Buoyant.jpg";
-
```

Comment:
These definitions were done outiside the component intentionally, as we don't want the imports to be inside the component and bloat up the code. Please undo this